impl/sound.cpp

```diff
@@ -115,4 +114,43 @@ void Sound::setPitch(float const newPitch)
     alSourcef(m_sourceId, AL_PITCH, newPitch);
 }
 
+void Sound::queueBuffer(ALuint buffer, std::size_t samplesToQueue)
+{
+    alBufferData(buffer, m_format, &m_soundData.getSamples().data()[m_cursor],
```

Comment:
.data() call can be removed
```suggestion
    alBufferData(buffer, m_format, &m_soundData.getSamples()[m_cursor],
```